src/services/email.service.js

```diff
@@ -0,0 +1,54 @@
+import 'dotenv/config';
+import nodemailer from 'nodemailer';
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASSWORD,
+  },
+});
+
+function send({ email, subject, html }) {
+  return transporter.sendMail({
+    to: email,
+    subject,
+    html,
+  });
```

Comment:
Consider adding error handling for the `transporter.sendMail` function to catch and log any errors that occur during the email sending process. This will help in diagnosing issues with email delivery.